tests/ops/functions/test_equal.py

```diff
@@ -1238,46 +1290,40 @@ def test_control_wires_comparison(self, wire1, wire2, res):
         assert qml.equal(op1, op2) == res
 
     @pytest.mark.parametrize(("wire1", "wire2", "res"), WIRES)
-    def test_work_wires_comparison(self, wire1, wire2, res):
+    def test_controlled_work_wires_comparison(self, wire1, wire2, res):
         """Test that equal compares work_wires for Controlled operators"""
         base1 = qml.MultiRZ(1.23, [0, 1])
         base2 = qml.MultiRZ(1.23, [0, 1])
         op1 = Controlled(base1, control_wires=2, work_wires=wire1)
         op2 = Controlled(base2, control_wires=2, work_wires=wire2)
         assert qml.equal(op1, op2) == res
 
-    def test_mismatched_arithmetic_depth(self):
-        """Test that comparing Controlled operators of mismatched arithmetic depth returns False"""
-        base1 = qml.PauliX(0)
-        base2 = qml.prod(qml.PauliX(0), qml.PauliY(1))
-        op1 = Controlled(base1, control_wires=2)
-        op2 = Controlled(base2, control_wires=2)
-
-        assert op1.arithmetic_depth == 1
-        assert op2.arithmetic_depth == 2
-        assert qml.equal(op1, op2) == False
-
-    def test_comparison_of_base_not_implemented_error(self):
-        """Test that comparing Controlled operators of base operators whose comparison is not yet implemented raises an error"""
-        base = qml.prod(qml.RX(1.2, 0), qml.RY(1.3, 1))
-        op1 = Controlled(base, control_wires=2)
-        op2 = Controlled(base, control_wires=2)
-
-        with pytest.raises(NotImplementedError, match="Unable to compare base operators "):
-            qml.equal(op1, op2)
-
-    @pytest.mark.torch
-    @pytest.mark.jax
-    def test_kwargs_for_base_operator_comparison(self):
-        """Test that setting kwargs check_interface and check_trainability are applied when comparing the base operators"""
-        import torch
-        import jax
-
-        base1 = qml.RX(torch.tensor(1.2), wires=0)
-        base2 = qml.RX(jax.numpy.array(1.2), wires=0)
-
-        op1 = Controlled(base1, control_wires=1)
-        op2 = Controlled(base2, control_wires=1)
+    @pytest.mark.parametrize("base", PARAMETRIZED_OPERATIONS)
+    def test_adjoint_comparison(self, base):
```

Comment:
Test docstrings seem to be missing here.

Reply:
Thanks, added now